src/main/scala/com/minosiants/pencil/Smtp.scala

```diff
@@ -347,8 +348,10 @@ object Smtp {
               ).run(req)
               _ <- readAll[F](a.file, req.blocker, 1024)
                 .through(fs2.text.base64.encode)
-                .evalMap { part =>
-                  lines[F](part).run(req)
+                .flatMap(s => Stream.chunk(Chunk.chars(s.toCharArray)))
```

Comment:
@BalmungSan this part is awesome. I didn't know how to chunk the stream. :thumbsup:

Could you extract it as standalone PR. I will merge it. I Still dont want to merge `Content-Disposition`

Reply:
Also could you remove this line since we dont need it anymore
```def lines[F[_]: Applicative](txt: String)```

Reply:
> Could you extract it as standalone PR. I will merge it. I Still dont want to merge Content-Disposition

Sure.

> Also could you remove this line since we dont need it anymore

`lines` is also used for the body.

Reply:
>lines is also used for the body.

Ok cool